scripts/Diffraction/isis_powder/pearl.py

```diff
@@ -80,8 +80,10 @@ def _apply_temporary_inst_settings(self, kwargs, run):
 
         # set temporary settings
         if not self._inst_settings.long_mode == bool(kwargs.get("long_mode")):
+            self._inst_settings.update_attributes(kwargs=kwargs)
```

Comment:
could we not do `self._inst_settings.update_attributes` before the if statement (i.e. do it regardless) and then keep the if statement as it is?

Reply:
we could, I was just dumb an missed that